bip-0340/speedup-batch/Makefile

```diff
@@ -0,0 +1,11 @@
+bench_output.txt: bench.sh
+	SECP256K1_BENCH_ITERS=100000 ./bench.sh bench_output.txt
+
+batch.dat: bench_output.txt
+	cat bench_output.txt | grep -v "schnorrsig_batch_verify_1" | awk 'match($$0, /schnorrsig_batch_verify_(.*):.*avg (.*)us /, a) {print a[1] " " a[2]}' > batch.dat
```

Comment:
This is ignoring all numbers where the batch size starts with decimal digit "1". By using `grep -v "schnorrsig_batch_verify_1:"` that is avoided.

Reply:
good catch, fixed